WebScanner-web/ClientApp/src/app/services/serverorders-backendservice.ts

```diff
@@ -3,15 +3,18 @@ import { Observable } from "rxjs/Observable";
 import { ServerOrder } from "../models/ServerOrder";
 import { ServerOrdersBackendServiceInterface } from '../services/serverorders-backendservice-interface';
 import { Http, RequestOptions, Headers, Response } from '@angular/http';
+import 'rxjs/add/operator/map';
 
 
 @Injectable()
 export class ServerOrdersBackendService extends ServerOrdersBackendServiceInterface {
-
-  private addServerOrderUrl: string = "NEED_TO_ADD";
-  private deleteServerOrderUrl: string = "NEED_TO_ADD";
-
+   
+  private addServerOrderUrl: string = "https://webscanner.ptrd.pl/api/serverorders";
+  private deleteServerOrderUrl: string = "https://webscanner.ptrd.pl/api/serverorders?orderId=";
+  private getServerOrderUrl: string = "https://webscanner.ptrd.pl/api/serverorders?orderId=";
+  private getAllServerOrdersUrl: string = "https://webscanner.ptrd.pl/api/serverorders/GetAll";
```

Comment:
Imo it should looks like this:
`"https://" + webscanner-host + "/api/serverorders/GetAll";`